### DISCUSSION TITLE
Fetching Branch

### DISCUSSION THREAD

**Discussion by Neodevils** [community/community#57623] [Mobile]
### Select Topic Area

Product Feedback

### Body

It would be great to fetching the branches via GitHub Mobile.

**Comment by HenSquared** [comment]
Hey @Neodevils , Do you mind clarifying the behavior you're hoping for?

You are able to see a list of branches on a repository

Navigate to a repo > scroll down until you see the name of the default branch. There you'll see a button 'change branch'. Selecting that will take you to a list of branches. You can select the branch and browse code in that branch.

If there's another behavior you're expecting, let me know. Thanks!

**Reply by Neodevils** [reply]
I was meant for branches behind of main branch for example. It would be good to implementing it on GitHub Mobile to fetch upstream to main branch.

**Reply by HenSquared** [reply]
Oh okay. I think I understand. 

If a branch is outdated, you would like to update the branch with the latest changes from main. 

Is my understanding correct?

You are able to do this from a Pull Request. 
On an outdated Pull Request, there is a button labeled 'Update Branch'. This will update the outdated branch with the latest changes from main.

**Reply by Neodevils** [reply]
> Oh okay. I think I understand.
> 
> If a branch is outdated, you would like to update the branch with the latest changes from main.
> 
> Is my understanding correct?
> 
> You are able to do this from a Pull Request. On an outdated Pull Request, there is a button labeled 'Update Branch'. This will update the outdated branch with the latest changes from main.

Not really. There is no option to upstream the branch to main branch on GitHub Mobile. I just checked an old branch and it is not showing any thing related to upstream to main branch.
